composer.json

```diff
@@ -36,11 +36,7 @@
     },
     "autoload": {
         "psr-4": {
-            "Yiisoft\\Db\\Pgsql\\": "src"
-        }
-    },
-    "autoload-dev": {
```

Comment:
Why did you delete this?

Reply:
Because I am going to run the tests from each implementation in db, it currently works the other way around, and the idea is to move the tests to each implementation you can read the issue.

Reply:
It's wrong, because if `db-mysql`'s tests are not valid that it means that `db-mysql` package implemented wrong, not `db` implemented.
Deleting this section will negatively affect to the autoloader.

Read the purpose https://getcomposer.org/doc/04-schema.md#autoload-dev

Reply:
The tests are not bad please check, remember db is the abstraction and db-mysql is the implementation, we currently have the code of both tests repeated, which is difficult to maintain, the idea is to have the tests in place, and unstoppable Each package, remember this package is large and must be done in parts.

Reply:
To explain it in db we will have the abstract classes of the tests, in each implementation the tests of each controller, then I will execute from db, for now the tests of the implementation that will verify all possible cases without duplicate code.